frontend/index.html

```diff
@@ -55,7 +55,7 @@ <h1 class="title">Outdated browser</h1>
       <section class="hero is-fullheight">
         <div class="hero-body">
           <div class="container has-text-centered">
-            <h1 class="title">JavaScript disabled!</h1>
+            <h1 class="title is-1">JavaScript disabled!</h1>
```

Comment:
In some places you add `is-1` and remove `title` to h1 elements. Here you just add `is-1`, while leaving `title`.

- Why do h1 elements need special classes at all?
- Should you remove `title` here like you did elsewhere? If not, should you add it to where you removed it?
- Why was `h1` removed from the `is-1` styling up above?

Reply:
Okay, I reviewed all the places. `.title` isn't needed at all (removed). Just `.is-[n]` is enough. I updated our DS page to explain a little more about it. 
See my other comment I added in this PR with a tricky situation.